bin/git-bug

```diff
@@ -1,14 +1,55 @@
 #!/usr/bin/env bash
 
-if test "$1" = "finish"; then
-  test -z $2 && echo "bug <name> required." 1>&2 && exit 1
-  branch=bug/$2
-  git merge --no-ff $branch && git delete-branch $branch
+branch_prefix=bug
+declare -a argv
+while test $# != 0
+do
+    case $1 in
+    -a|--alias )
+        if [[ -n $2 ]]
+        then
+            shift # shift -a|-alias
+            branch_prefix=$1
+        else
+            argv+=($1) # treat tail '-a' as <name>
+        fi
+        ;;
+	-r|--remote )
+		if [[ -n $2 ]]
+		then
+			remote=$2
+			shift
+		else
+			remote="origin"
+		fi
+		;;
+    * )
+        argv+=($1)
+        ;;
+    esac
+    shift
+done
+
+concatargs(){
+    str=$(IFS='-'; echo "$*")
+    branch="$branch_prefix"/$str
+}
+
+if test "${argv[0]}" = "finish"; then
+	test -z "${argv[1]}" && echo "$branch_prefix" "<name> required." 1>&2 && exit 1
+	branch="$branch_prefix"/"${argv[1]}"
+	git merge --no-ff "$branch" && git delete-branch "$branch"
 else
-  test -z $1 && echo "bug <name> required." 1>&2 && exit 1
-  if test -n "$2"; then
-    echo "too many arguments; if not finishing a bug, only <name> of new bug is expected" && exit 1
-  fi
-  branch=bug/$1
-  git checkout -b $branch
+	test -z "${argv[0]}" && echo "$branch_prefix" "<name> required." 1>&2 && exit 1
+	if test -n "${argv[1]}"; then
+		concatargs "${argv[@]}"
+	else 
+		branch="$branch_prefix"/"${argv[0]}"
+	fi
+	if [[ -n $remote ]]
+	then
+		git create-branch -r $remote $branch
+	else
+		git create-branch $branch
+	fi
 fi
```

Comment:
Wow! Now `git bug|chore|feature|refactor` are almost the same!
Shall we create a new inner command, and extract the common logic into it?
Just like [git-line-summary](https://github.com/tj/git-extras/blob/master/bin/git-line-summary) for `git summary --line).
It would be helpful for the future maintaining.